central/role/resources/list.go

```diff
@@ -24,56 +24,115 @@ import (
-	NetworkBaseline                  = newResourceMetadata("NetworkBaseline", permissions.NamespaceScope)
 	NetworkGraph                     = newResourceMetadata("NetworkGraph", permissions.NamespaceScope)
-	NetworkGraphConfig               = newResourceMetadata("NetworkGraphConfig", permissions.GlobalScope)
 	NetworkPolicy                    = newResourceMetadata("NetworkPolicy", permissions.NamespaceScope)
 	Node                             = newResourceMetadata("Node", permissions.ClusterScope)
-	Notifier                         = newResourceMetadata("Notifier", permissions.GlobalScope)
 	Policy                           = newResourceMetadata("Policy", permissions.GlobalScope)
-	ProbeUpload                      = newResourceMetadata("ProbeUpload", permissions.GlobalScope)
-	ProcessWhitelist                 = newResourceMetadata("ProcessWhitelist", permissions.NamespaceScope)
-	Risk                             = newResourceMetadata("Risk", permissions.NamespaceScope)
-	Role                             = newResourceMetadata("Role", permissions.GlobalScope)
-	ScannerBundle                    = newResourceMetadata("ScannerBundle", permissions.GlobalScope)
-	ScannerDefinitions               = newResourceMetadata("ScannerDefinitions", permissions.GlobalScope)
 	Secret                           = newResourceMetadata("Secret", permissions.NamespaceScope)
-	SensorUpgradeConfig              = newResourceMetadata("SensorUpgradeConfig", permissions.GlobalScope)
 	ServiceAccount                   = newResourceMetadata("ServiceAccount", permissions.NamespaceScope)
-	ServiceIdentity                  = newResourceMetadata("ServiceIdentity", permissions.GlobalScope)
-	SignatureIntegration             = newResourceMetadataWithFeatureFlag("SignatureIntegration", permissions.GlobalScope, features.ImageSignatureVerification)
-	User                             = newResourceMetadata("User", permissions.GlobalScope)
-	VulnerabilityManagementApprovals = newResourceMetadata("VulnerabilityManagementApprovals", permissions.GlobalScope)
-	VulnerabilityManagementRequests  = newResourceMetadata("VulnerabilityManagementRequests", permissions.GlobalScope)
-	VulnerabilityReports             = newResourceMetadata("VulnerabilityReports", permissions.GlobalScope)
-	WatchedImage                     = newResourceMetadata("WatchedImage", permissions.GlobalScope)
-
-	// Internal Resources
+	VulnerabilityManagementApprovals = newResourceMetadata("VulnerabilityManagementApprovals",
+		permissions.GlobalScope)
+	VulnerabilityManagementRequests = newResourceMetadata("VulnerabilityManagementRequests",
+		permissions.GlobalScope)
+	VulnerabilityReports = newResourceMetadata("VulnerabilityReports", permissions.GlobalScope)
+	WatchedImage         = newResourceMetadata("WatchedImage", permissions.GlobalScope)
+
+	// Deprecated resources.
+
+	// Deprecated: AllComments is deprecated, use Administration.
```

Comment:
Does it make sense to keep this comment for each resource, or does the use of `newResourceMetadataWithReplacingResourceMetadata` already gives this information ?
If the function name is not explicit enough to replace the comment, another more explicit name could be used.
If the function name is explicit enough, the deprecated resources could stay in the alphabetically sorted list too.

Reply:
The benefit I see from having these comments it to mark it within the IDE as deprecated, since clients won't notice when referencing those resources that they are deprecated. So its more of a cosmetic, IDE-related change.